src/query/service/src/interpreters/interpreter_insert.rs

```diff
@@ -309,6 +334,20 @@ impl InsertInterpreter {
                     let append_entries = ctx.consume_precommit_blocks();
                     // We must put the commit operation to global runtime, which will avoid the "dispatch dropped without returning error" in tower
                     GlobalIORuntime::instance().block_on(async move {
+                        let mut copied_files = BTreeMap::new();
+                        for file in need_copy_files {
+                            // Short the etag to 7 bytes for less space in metasrv.
```

Comment:
This doesn't seem safe.

Reply:
cc @youngsofun

Reply:
Can you convert the `insert ... attach file` to `copy`?

Reply:
copy will record the file metadata in the meta server.
insert_with_attach  is an internal optimization for insert, 
the file is temporary.

Reply:
i guess the insert ... attach file can share the same logic with the `copy` statement ?

Reply:
yes, but the detail depends on whether we  `need to record copied files`